### PR TITLE
Added: ShadCN Tabs Component with React Router NavLink

### PR DESCRIPTION
**Pull request by shashankvish0010**
### Description
Integrated ShadCN's Tabs component in the Home component, with functional "Statistics" and "Workflows" tabs.

fixes: #2265 

### Breaking changes
 * Changes are applied to Home, as both the statistics and workflows are the child element of Home.

### Examples and references
Screenshot (324)
Screenshot (325)

### CODE REVIEW

File: apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx [hunk 1]
```diff
@@ -21,7 +26,24 @@ export const Home: FunctionComponent = () => {
           {firstName && ` ${firstName}`}
         </h3>
       </div>
-      <Outlet />
+      <div>
+        <Tabs defaultValue="statistics">
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx]: The default value should be dependent on the URL. i.e. if the current route is `/:locale/workflows` then `defaultValue` should be `workflows`.

**Comment by Omri-Levy** [comment]
Note the "Home" nav item should be active whenever either of the tabs are selected. Both "Statistics" and "Workflows" in addition to `/:locale`.

File: apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
   const { data: session } = useAuthenticatedUserQuery();
   const { firstName, fullName, avatarUrl } = session?.user || {};
+  const isAuthenticated = useIsAuthenticated();
+  const { locale } = useParams();
+  const { pathname } = useLocation();
+  const navigate = useNavigate();
+  const value = pathname.includes('workflows') ? 'workflows' : 'statistics';
+
+  useEffect(() => {
+    if (isAuthenticated && pathname === `/${locale}`) {
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx]: No need to handle authentication here. It is covered by the layout.

File: services/workflows-service/src/workflow/cron/ongoing-monitoring.cron.ts
Comment (Omri-Levy) [inline services/workflows-service/src/workflow/cron/ongoing-monitoring.cron.ts:1]: Please make sure to not merge from dev unless you do the same for the epic branch first. 🙏🏼

File: apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
+  const { locale } = useParams();
+  const { pathname } = useLocation();
+  const navigate = useNavigate();
+  const value = pathname.includes('workflows') ? 'workflows' : 'statistics';
+
+  useEffect(() => {
+    if (pathname === `/${locale}`) {
+      navigate('/en/statistics');
+    }
+  }, [pathname]);
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx]: Early return on a falsy (!==, etc) tends to result in better readability. Once we decide to add more checks and logic doing so would also keep indentation levels low.
```suggestion
  useEffect(() => {
    if (pathname !== `/${locale}`) {
         return;
    }
    
    navigate('/en/statistics');
  }, [pathname]);
```

File: apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx [hunk 4]
```diff
@@ -1,15 +1,29 @@
...
   const { data: session } = useAuthenticatedUserQuery();
   const { firstName, fullName, avatarUrl } = session?.user || {};
+  const { locale } = useParams();
+  const { pathname } = useLocation();
+  const navigate = useNavigate();
+  const value = pathname.includes('workflows') ? 'workflows' : 'statistics';
+
+  useEffect(() => {
+    if (pathname === `/${locale}`) {
+      navigate('/en/statistics');
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx]:
```suggestion
      navigate(`/${locale}/statistics`);
```

File: apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx [hunk 5]
```diff
@@ -21,7 +35,21 @@ export const Home: FunctionComponent = () => {
...
+              <NavLink to={`/${locale}/statistics`}>Statistics</NavLink>
+            </TabsTrigger>
+            <TabsTrigger asChild={true} value="workflows">
+              <NavLink to={`/${locale}/workflows`}>Workflows</NavLink>
+            </TabsTrigger>
+          </TabsList>
+          <TabsContent value={value}>
+            <Outlet />
+          </TabsContent>
+        </Tabs>
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx]: What do you think?
```suggestion
        <Tabs defaultValue={value}>
          <TabsList>
            <TabsTrigger asChild={true} value={`/${locale}/statistics`}>
              <NavLink to={`/${locale}/statistics`}>Statistics</NavLink>
            </TabsTrigger>
            <TabsTrigger asChild={true} value={`/${locale}/workflows`}>
              <NavLink to={`/${locale}/workflows`}>Workflows</NavLink>
            </TabsTrigger>
          </TabsList>
          <TabsContent value={pathname}>
            <Outlet />
          </TabsContent>
        </Tabs>
```

File: apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx (hunk 3 above)
Comment (Omri-Levy) [inline apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx]: Note ESLint warns some items are missing from the dependency array.
```suggestion
  }, [locale, pathname, navigate]);
```

File: apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx (hunk 5 above)
Comment (shashankvish0010) [reply apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx]: I agree, as it's now more explicit and easier to understand.

**Review by tomer-shvadron — APPROVED** [review]

**Pull request merged**